tests/System.Text.Encodings.Web.Utf8.Tests/UrlEncoderTests.cs

```diff
@@ -87,10 +88,12 @@ public abstract class UrlEncoderTests
         [InlineData("%C2%B5%40%C3%9F%C3%B6%C3%A4%C3%BC%C3%A0%C3%A1", "µ@ßöäüàá")]
         [InlineData("%C2%B5%40%C3%9F%C3%B6%C3%A4%C3%BC%C3%A0%C3%A", "µ@ßöäüà%C3%A")]
         public void DecodeWithBoundary(string raw, string expect) => TestCore(raw, expect);
+    }
 
-        protected abstract void TestCore(string raw, string expected);
+    public abstract class UrlCoderTests
```

Comment:
nit: new file
Also, what does UrlCoder mean?

Reply:
It's meant for code that is common to encoder and decoder